section3/reflection.md

```diff
@@ -2,16 +2,32 @@
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
 
+I feel the same - I typically work using this technique and so I was just more mindful but each time I took organic rests it was following this technique.
+
+
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
 
+They were pretty spot on for me!
+
 3. What is a Hash, and how is it different from an Array?
 
+A Hash will support any object as a key whereas an Array will use an integer
+
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
 
+pet_store = ['toys':10, 'food':8, 'animals':4]
+
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
 
+states[‘IA’]
+
 6. With the same hash above, how would we get all the keys?  How about all the values?
 
+states.each { | key, value | puts key}
+
 7. What is another example of when we might use a hash?  In your example, why is a hash better than an array?
 
-8. What questions do you still have about hashes?
\ No newline at end of file
+The number of cars on a lot. Many cars will have different attributes and many will have the same. Using a hash will allow us to use the objects a key instead of the integer.
+
+8. What questions do you still have about hashes?
+How often are hashes used?
```

Comment:
All the time!!! They are a very useful datatype because they allow a "lookup" that is FAST! 
(aka, it's much faster to find something in a hash with 1 million key/value pairs than to find the same thing in an array with 1 million elements)